_episodes/07-cond.md

```diff
@@ -53,7 +53,8 @@ If the test is false,
 the body of the `else` is executed instead.
 Only one or the other is ever executed:
 
-![Executing a Conditional](../fig/python-flowchart-conditional.png)
+![If-Else flow diagram for num variable testin if it is greather than 100.
+If true it prints 'greater' and 'done', if false it prints 'not greather' and 'done'.](../fig/python-flowchart-conditional.png)
```

Comment:
Another possibility to consider is modifying the text before the figure slightly in order to reduce the `alt` attribute text. For example:

```
The second line of this code uses the keyword `if` to tell Python that we want to make a choice.
If the test that follows the `if` statement is true,
the body of the `if`
(i.e., the set of lines indented underneath it) is executed, and "greater" is printed.
If the test is false,
the body of the `else` is executed instead, and "not greater" is printed.
Only one or the other is ever executed before continuing on with program execution to print "done":

![If-Else flow diagram testing if variable num is greater than 100]
```

Reply:
good point, changed to that.